lib/path-set.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * A unique set of paths. Case insensitive on win32
+ */
+class PathSet extends Set {
+  constructor () {
+    super()
+    this._win32 = process.platform === 'win32'
+  }
+
+  add (value) {
+    if (this._win32) {
+      value = value.toLowerCase()
```

Comment:
My concern is that I think Windows can have case sensitivity enabled? [see this article](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc725747(v=ws.11)?redirectedfrom=MSDN).

I wonder if we could find a way to feature detect case sensitivity, rather than basing it on Windows?

Reply:
Oh I didn't know that. Will review. Thanks!

Reply:
I looked into that for jest and didn't find a nice way (suggested solutions was writing a file and reading it with different casing, but it had some caveats).

If you _do_ find a nice way, please publish it as a separate module and we can use it in Jest as well 😀

Reply:
@SimenB did you publish a module for the approach used by Jest? otherwise I'd be happy to pull together a module that at least exposes the hack.

Reply:
No, we gave up and just check for `platform === 'win32'` 😞 So please put together a module! Would be cool if Node itself could expose something...